<think>
No reasoning found</think>
Here is the code snippet with the merge conflict preserved:

```java
import org.apache.pig.backend.executionengine.ExecException;
import org.apache.pig.backend.hadoop.datastorage.ConfigurationUtil;
import org.apache.pig.data.DataType;
import org.apache.pig.data.Tuple;
import org.apache.pig.impl.PigContext;
<<<<<<< HEAD
import org.apache.pig.impl.io.FileLocalizer;
=======
import org.apache.pig.impl.PigImplConstants;
>>>>>>> 2125666c1941d4a88c08ab87ebf92ee46be4f0ea
import org.apache.pig.impl.io.InterStorage;
import org.apache.pig.impl.io.ReadToEndLoader;
import org.apache.pig.impl.io.SequenceFileInterStorage;
import org.apache.pig.impl.io.TFileStorage;
import org.apache.pig.impl.logicalLayer.schema.Schema;
```